src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/featured-image.php

```diff
@@ -27,14 +27,18 @@
 		rel="bookmark"
 		class="tribe-events-calendar-month__calendar-event-tooltip-featured-image-link"
 	>
-		<div class="tribe-events-calendar-month__calendar-event-tooltip-featured-image tribe-common-c-image tribe-common-c-image--bg">
-			<div
-				class="tribe-common-c-image__bg"
-				style="background-image: url('<?php echo esc_url( $event->thumbnail->full->url ); ?>');"
-				role="img"
-				aria-label="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
-			>
-			</div>
-		</div>
+		<img
+			src="<?php echo esc_attr( $event->thumbnail->full->url ); ?>"
```

Comment:
:no_entry_sign: Wrong escaping function. href and src attributes should be escaped by `esc_url()`, not by `esc_attr()`.

<sub>hash: 4f415454083280d4ce9c27ddd41bf014</sub>